README.md

```diff
@@ -8,50 +8,23 @@
 
 [Renovate sharable config](https://docs.renovatebot.com/config-presets/) for [.tool-versions (asdf)](https://github.com/asdf-vm/asdf-plugins/tree/master/plugins).
 
+Since renovate version [34.25.0](https://github.com/renovatebot/renovate/pull/18612#issuecomment-1315123140), official asdf manager imported most plugins of this repository.\
+So you don't need to depend this config excepts hugo/gohugo.\
+This repository's hugo definition supports [extended hugo releases](https://github.com/gohugoio/hugo/blob/af23cdca9c9c230ffbffbab96f9600a78c76b75f/docs/content/en/troubleshooting/faq.md?plain=1#L50-L60). See the [issue](https://github.com/kachick/renovate-config-asdf/issues/294) for further detail.\
```

Comment:
@kachick This will be resolved with https://github.com/renovatebot/renovate/pull/18987#pullrequestreview-1186547153

Reply:
Oh... Thanks for your polite work! 🙏

Reply:
@kachick My pleasure :) Thanks for coming up with the basis for all this!